tests/276_Imagick_artifacts.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+Test artifacts
+--SKIPIF--
+<?php
+
+require_once(dirname(__FILE__) . '/skipif.inc');
+checkClassMethods('Imagick', array('setImageArtifact', 'getImageArtifact', 'deleteImageArtifact'));
+
+/* See imagick_class.c */
+if (Imagick::getVersion()['versionNumber'] < 0x656) {
```

Comment:
That line isn't quite valid syntax on PHP 5.3 - https://3v4l.org/j57pb

but the test passes for most of the PHP 5.3 versions. Which is interesting.